apps/frontline-widgets/src/formBundle.js

```
@@ -1,268 +1,454 @@
+      if (!o || d !== 'fromForms') return null;
+      let { container: i, iframe: s } = f[v(o)] || {};
+      P(e, s);
+      let a = R(o);
+      if (!a || !(n && d === 'fromForms')) return null;
+      if (
+        (r === 'submitResponse' && a.onAction && a.onAction(t),
+        r === 'connected' &&
+          t.connectionInfo.widgetsLeadConnect.form.leadData.loadType ===
+            'popup')
+      ) {
+        let E = `[data-erxes-modal="${o.form_id}"]`,
```

ESLint

[error] 'E' is never reassigned. Use 'const' instead.

(prefer-const)